routes/chat.js

```diff
@@ -6,11 +6,13 @@ const chatController = require('../controllers/chat/ChatController');
 const auth = require('../middlewares/auth');
 const isTargetUserAllowingAnonDMMiddleware = require('../middlewares/chat/isTargetUserAllowingAnonDMMiddleware');
 const SuccessMiddleware = require('../middlewares/success');
+const sendSignedMesage = require('../controllers/chat/sendSignedMessage');
 
 router.get('/create-channel', chatController.createChannel);
 router.post('/add-moderator', chatController.addChannelModerator);
 router.post('/add-members-channel', auth.isAuth, chatController.addMembers);
-router.post('/anonymous', auth.isAuth, chatController.sendAnonymous);
+router.post('/anonymous', auth.isAuthAnonim, chatController.sendAnonymous);
+router.post('/send-signed-message', auth.isAuthV2, sendSignedMesage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/send-signed-message` is added with `auth.isAuthV2` middleware. This seems to be a good practice as it ensures that only authenticated users can send signed messages. However, please ensure that the `auth.isAuthV2` middleware correctly authenticates and authorizes the user before allowing them to send a signed message.
```diff
+ const sendSignedMesage = require('../controllers/chat/sendSignedMessage');
+ router.post('/send-signed-message', auth.isAuthV2, sendSignedMesage);
```
Also, there's a minor typo in the name of the imported function `sendSignedMesage`. It should be `sendSignedMessage`.
```diff
- const sendSignedMesage = require('../controllers/chat/sendSignedMessage');
+ const sendSignedMessage = require('../controllers/chat/sendSignedMessage');
- router.post('/send-signed-message', auth.isAuthV2, sendSignedMesage);
+ router.post('/send-signed-message', auth.isAuthV2, sendSignedMessage);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->